travis/stage-python3.sh

```diff
@@ -2,7 +2,6 @@
 set -ex
 if ! [[ "$TRAVIS_EVENT_TYPE" == "cron" || "$TRAVIS_BUILD_STAGE_NAME" == "Nightly" ]]
 then
-  ./travis/run-large-python-tests.sh
```

Comment:
Just to make sure, do we also plan to move the anaconda compatibility tests to github actions? It's actually good that this PR is scoped & focuses specifically on the large python tests, just wanted to check - thanks!

Reply:
Yes, I'm planning to move all the tests to GitHub Actions. For easier review and debug, I'll do one by one.